web_src/js/features/repo-issue-list.js

```diff
@@ -194,6 +194,10 @@ function initArchivedLabelFilter() {
   const url = new URL(window.location.href);
   const archivedLabels = document.querySelectorAll('[data-is-archived]');
 
+  if (!archivedLabels.length) {
+    hideElem($('.archived-label-filter'));
```

Comment:
```suggestion
    hideElem($('.archived-label-filter'));
```
for shorthand I have used `$` Jquery selector.

Reply:
The jQuery is not necessary for `hideElem`.

```
hideElem('.archived-label-filter');
```

Reply:
This is cool